lib/json.coffee

```diff
@@ -0,0 +1,74 @@
+module.exports =
+  parse: (graph, options = {}) ->
```

Comment:
Please let the parse function be at the first (no) indentation level. Then at the end do the `module.export = ...`